wordpress-mobile/MediaPicker-iOS
# Fix accessibility on picker

The last changes on the picker to add placeholders for some assets and in error scenarios made the accessibility information being set incorrectly.

This PR address the issue.

To test:
 * Open the picker on the simulator
 * Open the accessibility inspector on XCode
 * Using the inspector check that the accessibility information on cells shows correctly.
